lib/pos_mempool.go

```diff
@@ -118,6 +121,14 @@ type PosMempool struct {
 	// PosMempool only needs read-access to the block view. It isn't necessary to copy the block view before passing it
 	// to the mempool.
 	readOnlyLatestBlockView *UtxoView
```

Comment:
A minor note to help you understand my other comments: This field is *truly* readOnly now, meaning that we don't ever modify the underlying view after we set it. Instead, we just change the *pointer* to point to a new view. This means that the code for updating this value, as well as the augmentedLatestBlockView value, is *much* more efficient than it was with the legacy mempool. The reason is that the legacy mempool used to require a view-copy to be inside of a lock, whereas we only need a pointer-copy to be inside of a lock. Much better than before! Which means we can have a lower time between each update. I recommend 10ms.